torchtext/datasets/conll2000chunking.py

```diff
@@ -45,20 +45,27 @@ def CoNLL2000Chunking(root: str, split: Union[Tuple[str], str]):
     url_dp = IterableWrapper([URL[split]])
 
     # Cache and check HTTP response
-    cache_dp = url_dp.on_disk_cache(
-        filepath_fn=lambda x: os.path.join(root, "conll2000chunking", os.path.basename(URL[split])),
-        hash_dict={os.path.join(root, "conll2000chunking", os.path.basename(URL[split])): MD5[split]},
```

Comment:
Thanks for removing extra folder from the path. I think this would address the issue here https://github.com/pytorch/text/pull/1570#discussion_r800782606